src/js/PostRendering.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PostTemplate from "./PostTemplate";
+import styled from "styled-components";
+import axios from "axios";
+
+const AllPostsContainer = styled.div `
+  width: 70%;
+  box-shadow: 7px 6px #dad5cb;
+  border: 1px solid black;
+  margin: 0 auto;
+  font-family: "Roboto", sans-serif;
+
+  @media (max-width: 700px) {
+    width: 95%;
+    padding: 0 4px;
+  }
+`;
+
+class BuildPosts extends React.Component {
```

Comment:
Why not import `{ Component }` from react at the top? You've done it in every other file in this PR

Reply:
wasn't my code, but done

Reply:
Thanks for changing it. It is not an excuse when it's your PR into a separate project.